RELEASE_NOTES.md

```diff
@@ -1,3 +1,18 @@
+# Version 0.5.9
+
+* Issue #12427 Update ADCP Parsers and Consolidate ADCP stream:
+    * stream "adcp_velocity_earth" replaces:
+        * adcp_velocity_glider
+        * adcp_velocity_inst
+    * stream "adcp_velocity_beam" replaces:
+        * adcp_pd0_beam_parsed
+        * vadcp_5thbeam_pd0_beam_parsed
+        * vadcp_pd0_beam_parsed
+    * stream "adcp_system_configuration" replaces:
+        * vadcp_4beam_system_configuration
+    * stream "adcp_system_configuration_5" replaces:
+        * vadcp_5thbeam_system_configuration
+        
```

Comment:
We probably should identify all the parsers that have been updated so the Data Team will know which parsers need to be rerun.

Reply:
its already attached to the ticket, like which parser should rerun

Reply:
As a courtesy to the Data Team, that will be looking at the RELEASE_NOTES.md file, not the pull request, can you please add in the affected streams.